tasks/agent.py

```diff
@@ -666,7 +667,22 @@ def bundle_install_omnibus(ctx, gem_path=None, env=None):
         cmd = "bundle install"
         if gem_path:
             cmd += f" --path {gem_path}"
-        ctx.run(cmd, env=env)
+
+        for trial in range(MAX_TRY_BUNDLE_INSTALL):
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏What do you think of having `MAX_TRY_BUNDLE_INSTALL` be a parameter to the function instead of a constant ?